project/Build.scala

```diff
@@ -578,7 +578,8 @@ object Build {
   lazy val commonDottyCompilerSettings = Seq(
        // Note: bench/profiles/projects.yml should be updated accordingly.
        Compile / scalacOptions ++= Seq("-Yexplicit-nulls", "-Ysafe-init"),
-
+      // Use source 3.3 to avoid fatal migration warnings on scalajs-ir
+      scalacOptions ++= Seq("-source", "3.3"),
```

Comment:
@sjrd where can I add this options to enable them only when compiling the `scalajs-ir`? 
See https://github.com/lampepfl/dotty/actions/runs/6784134103/job/18439824856

Reply:
I don't think you can. The `scalajs-ir` sources are built as part of `scala3-compiler` itself, so they must share their flags.